Cherubln/My-Brand-Cherubin
# #024 testing endpoints

#### What does this PR do?
it tests all endpoints
#### Description of Task to be completed
add testing package (mocha, chai, coveralls)
test all endpoints
#### How should this be manually tested?
git clone the repo
run npm install
 run npm  test
or run npm run coverage to see the coverage

#### Screenshots (if appropriate)

![screenshots](https://user-images.githubusercontent.com/50265203/90790833-c904cd00-e308-11ea-98a9-ce2a16b88e35.png)
